proto/buf.gen.swagger.yaml

```
@@ -11,4 +11,4 @@ plugins:
       - logtostderr=true
       - openapi_naming_strategy=fqn
       - json_names_for_fields=false
-      - generate_unbound_methods=true
+      - generate_unbound_methods=true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)